tests/checkPot.py

```diff
@@ -98,6 +97,24 @@
 	'Primary footer',
 	'Primary header',
 	'Text frame',
+	# core.py:87
+	r'Your gesture map file contains errors.\n"More details about the errors can be found in the log file."',

```

Comment:
Could you create an issue for these missing translator comment strings? This could be fairly straightforward work for a new contributor.

Reply:
Done: #11104 
Could you please add the label "goodForNewDev"?
(Or allow me to label issues...)